src/styles/LandingPageStyles/index.less

```diff
@@ -13,17 +13,14 @@
   background-image: url('../../img/cloud-bg.jpg');
 }
 
-.landing-main-text {
+.landing-main-text, .landing-second-container-text {
   margin: 5% 0%;
   text-align: center;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
I would import the font-families from the theme file in the style folder!

Reply:
Definitely will implement the change to match the theme.